src/pages/Integrations/SplunkSetup/SplunkSetupForm.tsx

```diff
@@ -157,3 +209,22 @@ const SplunkAutomationButton = ({ onStart, onFinish, stepIsInProgress, stepVaria
         return <Button variant="primary" onClick={ onStart }>Run Configuration</Button>;
```

Comment:
Haven't seen the design, but shouldn't this button be disabled if there is any error in the form? 
```suggestion
        return <Button 
            variant="primary" 
            isDisabled={ validatedHecToken !== ValidatedOptions.success || validatedServerHostname !== ValidatedOptions.success  } 
            onClick={ onStart }
        >
            Run Configuration
        </Button>;
```

Reply:
And also if it is in progress

Reply:
Indeed it should. Thank you!

The in progress one is not disabled. Here is the button from the mock:
![Screenshot 2022-04-26 at 10-02-12 Insights Splunk Integration](https://user-images.githubusercontent.com/7695766/165252045-a9f83354-0281-42f5-a790-ab8f33a3882e.png)

Reply:
Updated for the invalid (and default) state. Thank you!